packages/next/src/build/utils.ts

```diff
@@ -824,9 +823,10 @@ export function printCustomRoutes({
       })
       .join('\n')
 
-    print(routesStr, '\n')
+    print(`${routesStr}\n`)
   }
 
+  print()
```

Comment:
Ensure two line breaks (marked as `>` below) for each option:

```sh
Headers
┌ source: /
└ headers:
  └ x-custom-headers: headers
>
>
Rewrites
┌ source: /rewrites
└ destination: /
```

Reply:
@huozhi, sorry to ping you, but wanted to ask you whether two line breaks are OK since I believe you have good aesthetics.